sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -970,6 +986,21 @@ public PDone apply(PCollection<TableRow> input) {
         return VoidCoder.of();
       }
 
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        if (table != null) {
+          builder.add("tableSpec", toTableSpec(table));
+        }
```

Comment:
if `tableSpecFn` is present, probably worth adding `table function: tableSpecFn.getClass().getSimpleName()`.